hello-world/README.md

```diff
@@ -0,0 +1,64 @@
+## Conclave Sample
+
+This is a sample to demonstrate the use of Persistent Filesystem and Persistent Map in your enclave. The Persistent
+Filesystem can be used to persist files and make them available even during enclave restarts. On the other hand, the
+persistent map is a persistent, encrypted key-value store that can be used to store information from your enclave. The
+enclave can use this key-value store to persist information as required. The enclave of this Conclave sample does three
```

Comment:
This section essentially says they both persist data! You need to summarise the differences between the two. The new persistence page on the docsite should help you with that: https://docs.conclave.net/persistence.html.

Reply:
Also, the last sentence needs to be in a separate paragraph.